### PR TITLE
Card

### PR DESCRIPTION
**Pull request by parvez-rashid**
View the screenshots below:

image
image
image

### CODE REVIEW

File: card/card.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit officia
+        totam iure, omnis dolorem reprehenderit cupiditate possimus provident
+        harum, nulla minima hic, ratione aspernatur odit esse assumenda iste
+        beatae asperiores perspiciatis ad impedit labore. Porro hic sed
+        perferendis magnam animi inventore?
+      </p>
+    </div>
+    <div class="card vertical-card box-shadow border-rad-md">
+      <img class="border-rad-md" src="/card/src/image.jpg" alt="demo image" />
```
Comment (Naman-Saxena1) [inline card/card.html]: Maybe use more descriptive name for alt attribute for accessibility purpose.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: card/card.html (hunk 1 above)
Comment (parvez-rashid) [reply card/card.html:42]: Thanks, sure will do this.

Comment (parvez-rashid) [reply card/card.html]: Have updated the image description, please approve PR. Thanks.

**Review by Naman-Saxena1 — APPROVED** [review]
Looks great 👍

File: card/card.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/card/card.css" />
+    <script
+      src="https://kit.fontawesome.com/43943f2889.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
```
Comment (zabihshaik) [inline card/card.html:12]: Better to add title, for eg. Card | your CL Name

File: card/card.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/card/card.css" />
```
Comment (zabihshaik) [inline card/card.html:7]: Better to add meta tag with your name

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a few comments. Plz check.

File: card/card.html (hunk 3 above)
Comment (parvez-rashid) [reply card/card.html:7]: Okay, I'll look into it.

File: card/card.html (hunk 2 above)
Comment (parvez-rashid) [reply card/card.html:12]: Yes, I'll do this in the refactor.

**Pull request merged**